install-agent.sh

```diff
@@ -144,21 +136,16 @@ function install_trento() {
         systemctl stop trento-agent
     fi
 
-    if [[ -n "$USE_TGZ" ]]; then
-        echo "* Downloading trento tar.gz from GitHub..."
-        install_trento_tgz
-    else
-        install_trento_rpm
-    fi
+    install_trento_rpm
 }
 
 function install_trento_rpm() {
     if [[ -n "$USE_ROLLING" ]]; then
-        TRENTO_REPO=${TRENTO_REPO:-"https://download.opensuse.org/repositories/devel:/sap:/trento:/factory/15.3/devel:sap:trento:factory.repo"}
-        TRENTO_REPO_KEY=${TRENTO_REPO_KEY:-"https://download.opensuse.org/repositories/devel:/sap:/trento:/factory/15.3/repodata/repomd.xml.key"}
+        TRENTO_REPO=${TRENTO_REPO:-"https://download.opensuse.org/repositories/devel:/sap:/trento:/factory/standard/devel:sap:trento:factory.repo"}
+        TRENTO_REPO_KEY=${TRENTO_REPO_KEY:-"https://download.opensuse.org/repositories/devel:/sap:/trento:/factory/standard/repodata/repomd.xml.key"}
     else
-        TRENTO_REPO=${TRENTO_REPO:-"https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/devel:sap:trento.repo"}
-        TRENTO_REPO_KEY=${TRENTO_REPO_KEY:-"https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/repodata/repomd.xml.key"}
```

Comment:
Note the the reference here is no longer hard coded to 15.3, which was quite an oversight.
The OBS project now provides a "standard" repo, which is an alias of the latest SP (currently SP6).
See https://build.opensuse.org/projects/devel:sap:trento:factory/meta for more details.